docs/reference/field_types.rst

```diff
@@ -60,6 +60,52 @@ This is currently limited to scalar types (text, integer, url...) and choice typ
 More types might be provided based on the persistency layer defined. Please refer to their
 related documentations.
 
+Array
+^^^^^^
+
+You can use the following parameters:
+
+======================================  ============================================================
+Parameter                               Description
+======================================  ============================================================
+**inline**                              If `true` array will be displayed as a single line,
+                                        each element will be wrapped up with square brackets.
```

Comment:
Maybe the doc should be updated since the `inline` array now display `[key=>value, key=>value]`

Reply:
Good point, thanks!